conans/client/installer.py

```diff
@@ -316,16 +317,38 @@ def _node_concurrently_installed(self, node, package_folder):
             if node.update_manifest == read_manifest:
                 return True
 
+    def _load_editables_cpp_info(self):
+        editables_path = self._client_cache.default_editable_path
+        if os.path.exists(editables_path):
+            return EditableCppInfo.create(editables_path, require_namespace=True)
```

Comment:
``required_namespace`` maybe not very intuitive. I guess that means that repository layout files will error if they find some [LibA:includedirs] line. Maybe ``allow_package_prefix``?